### PR TITLE
fix: Refactor messages API and session view

### PR DESCRIPTION
**Pull request by anovazzi1**
This pull request includes several changes to the messages API and the session view component.

The changes include:

- Refactoring the messages API to use separate files get messages.

- Updating the session view component to use the new messages API.

- Removing unused code related to the messages table.

- Adding a loading indicator to the session view component.

These changes improve the organization and performance of the messages API and enhance the user experience in the session view component.

### CODE REVIEW

File: src/frontend/src/controllers/API/index.ts [hunk 1]
```diff
@@ -1071,31 +1071,6 @@ export async function getTransactionTable(
...
-  const sessions = new Set<string>();
-  rowsOrganized.forEach((row) => {
-    sessions.add(row.session_id);
-  });
-  return { rows: rowsOrganized, columns };
-}
-
 export async function deleteMessagesFn(ids: string[]) {
   try {
     return await api.delete(`${BASE_URL_API}monitor/messages`, {
```
Comment (ogabrielluiz) [inline src/frontend/src/controllers/API/index.ts:1076]: Should you use the constant here?

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
yo! 

Apart from not using the defined constant this looks good.

**Review by Cristhianzl — APPROVED** [review]
LGTM

File: src/frontend/src/controllers/API/index.ts (hunk 1 above)
Comment (anovazzi1) [reply src/frontend/src/controllers/API/index.ts:1076]: this delete endpoint is still on going

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**